tests/__init__.py

```diff
@@ -332,8 +332,8 @@ class PortalMock:
     Parent class for portal tests.
     """
 
-    def __init__(self, session_bus, portal_name: str, app_id: str = "org.example.App"):
-        self.bus = session_bus
+    def __init__(self, dbus_test_case, portal_name: str, app_id: str = "org.example.App"):
+        self.test_case = dbus_test_case
```

Comment:
arguably, it's better to use `self.dbus_test_case` here to avoid confusion with the individual actual test case we're running.